src/main/java/com/conveyal/analysis/components/broker/RedeliveryTest.java

```diff
@@ -64,9 +67,11 @@ private static void sendFakeJob(Broker broker) {
         templateTask.height = 1;
         templateTask.width = N_TASKS_PER_JOB;
         templateTask.scenarioId = "FAKE";
-        RegionalAnalysis regionalAnalysis = new RegionalAnalysis();
+        RegionalAnalysis regionalAnalysis = new RegionalAnalysis(new UserPermissions("testing@conveyal.com", false, "testing"), "test");
         regionalAnalysis.request = templateTask;
-        broker.enqueueTasksForRegionalJob(regionalAnalysis);
+        var job = new Job(templateTask, WorkerTags.fromRegionalAnalysis(regionalAnalysis));
+        var assembler = new MultiOriginAssembler(job, new ArrayList<>());
```

Comment:
Yes. Although I haven't looked at our style guide in ages. My short opinion / addition to the style guide is: prefer using `var` for all non-primitive types except in cases where we want to explicitly show a type. 

In the examples above, writing the types instead of `var` is redundant information on the same line: `Job`, and `MultiOriginAssembler` are already there.

A case where we might want to be more explicit, is when a method returns a value and we want to distinguish that type in relation to neighboring types.